### PR TITLE
pre-commit: Enable black --skip-string-normalization

### PR DESCRIPTION
**Pull request by cclauss**
Pre-commit is now fully enabled with the addition of https://github.com/psf/black

### CODE REVIEW

File: motioneye/config.py [hunk 1]
```diff
@@ -426,7 +488,9 @@ def get_camera(camera_id, as_lines=False):
         _set_default_simple_mjpeg_camera(camera_id, camera_config)
 
     else:  # incomplete configuration
-        logging.warning('camera config file at %s is incomplete, ignoring' % camera_config_path)
+        logging.warning(
+            'camera config file at %s is incomplete, ignoring' % camera_config_path
```
Comment (MichaIng) [inline motioneye/config.py:492]: I wonder why this has not become:
```py
f'camera config file at {camera_config_path} is incomplete, ignoring'
```
via #2348. Same in other cases here where the `%` variable injection is still used 🤔.

**Comment by MichaIng** [comment]
This will cause a lot of conflicts with open PRs. Probably we should wait with it until most of them have been merged.

File: motioneye/config.py (hunk 1 above)
Comment (cclauss) [reply motioneye/config.py:492]: See pyupgrade rules.

Comment (MichaIng) [reply motioneye/config.py:492]: The rule doesn't give more details about when if is skipped:
- First step: https://github.com/asottile/pyupgrade#printf-style-string-formatting
- Python 3.6+: https://github.com/asottile/pyupgrade#f-strings

Examples are only with at least two variables, but single variable strings have been converted as well, which makes sense for consistency.

**Comment by MichaIng** [comment]
Is it possible to get a diff from pre-commit failures? I removed this leftover from a resolved conflict within the imports in `sendtelegram.py` but isort is not happy with it:
```py
import datetime
import io
import logging
import os
import pycurl
import re
import signal
import socket
import sys
import time

from tornado.ioloop import IOLoop

from motioneye import config, mediafiles, motionctl, settings, utils
from motioneye.controls import tzctl

messages = {
    'motion_start': 'Motion has been detected by camera "%(camera)s/%(hostname)s" at %(moment)s (%(timezone)s).'
}
```
Complete imports, empty line, partial imports. empty line, internal imports, empty line, variable declaration, all alphabetically sorted. Not sure what's wrong 😅.

**Review by MichaIng — APPROVED** [review]
Okay, with #2110 the PR I was most worried about regarding conflicts has been merged, and I rebased and resolved the conflicts here. Ready to be merge from my end. It adds a large number of new lines to maintain a max 88 character line length, but it's good to have some consistency enforced with multiple developers.

**Review by MarcelCoding — APPROVED** [review]

**Pull request merged**